curvefs/src/metaserver/metaserver.cpp

```diff
@@ -580,6 +567,18 @@ void Metaserver::InitCopysetNodeOptions() {
 
     CHECK(localFileSystem_);
     copysetNodeOptions_.localFileSystem = localFileSystem_.get();
+
+    std::string dataDir;
+    std::string protocol = curve::common::UriParser::ParseUri(
+        copysetNodeOptions_.dataUri, &dataDir);
+
+    LOG_IF(ERROR, dataDir.empty())
+        << "Unsupported data uri: " << copysetNodeOptions_.dataUri;
+
+    LOG_IF(ERROR,
```

Comment:
Heartbeat will report metastore's disk and memory usage, previously, the storage data directory was independent of the copyset data directory and was created when creating the docker image. Now, we put each storage data directory under copysets, so we had to create copyset directory first, otherwise, the heartbeat will report an error.